src/Lexers/ModelLexer.php

```diff
@@ -82,6 +82,69 @@ class ModelLexer implements Lexer
         'unique' => 'unique',
     ];
 
+    private static $phpDocDataTypes = [
+        'id' => 'integer',
```

Comment:
This should be `int` not `integer` - see [Types](https://docs.phpdoc.org/latest/guides/types.html)